### PR TITLE
Generatebundle support long repos

### PR DESCRIPTION
**Pull request by jonahjon**
*Issue #, if available:*

*Description of changes:*

Can support repo names with multiple slashes in the names.    `charts/org/repo`

### CODE REVIEW

File: generatebundlefile/ecr_helper.go [hunk 1]
```diff
@@ -73,13 +73,19 @@ func splitECRName(s string) (string, string, error) {
 		if len(chartNameList) == 4 {
 			return fmt.Sprintf("%s/%s", chartNameList[2], chartNameList[3]), chartNameList[3], nil
 		}
```
Comment (ivyostosh) [inline generatebundlefile/ecr_helper.go:75]: I wonder if ==4 and > 4 scenario can be consolidated to keep the logic clean? I don't see issues with it but let me know if I omit something.

File: generatebundlefile/ecr_helper.go [hunk 2]
```diff
@@ -73,13 +73,19 @@ func splitECRName(s string) (string, string, error) {
...
+		if len(chartNameList) > 4 {
+			return strings.Join(chartNameList[2:], "/"), chartNameList[len(chartNameList)-1], nil
+		}
 	}
 	if len(chartNameList) == 2 {
 		return chartNameList[1], chartNameList[1], nil
 	}
 	if len(chartNameList) == 3 {
```
Comment (ivyostosh) [inline generatebundlefile/ecr_helper.go:83]: Same comment here for ==3 and >3 scenario. Can they be consolidated?

Comment (jonahjon) [reply generatebundlefile/ecr_helper.go:83]: Yea lemme test that out

Comment (jonahjon) [reply generatebundlefile/ecr_helper.go]: ok pushed with c930afcdf3ffeaafdbfc746c124542c2067710b3

**Pull request merged**